11-Component-assignments-3/src/Main.js

```diff
@@ -3,6 +3,8 @@ import ReactDOM from 'react-dom';
 
 import Guess from './Guess'
 
+
+// Why did you not save "target" and "cheating" in state?
```

Comment:
good question. I think i just don't consider them changeable during the run of the app. once the app is running, the target and cheating factor are pre-defined the way I wrote this. preventing changes during "game". but they can 100% be states, but i don't see where i would use "setTarget" and "setCheating" other then at the start of the app. maybe if i added a new game or a restart button to allow repeatability.